File: UF1/M3-UF1-practica-5/ejercicio11.java
import java.io.*;
 
public class ejercicio11{
	
  public static void main(String[] args) throws IOException {
    float A;
    BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
    System.out.print("Introduce el precio del producto: ");
    String linea = reader.readLine();
    A = Integer.parseInt(linea);  // conversió de String a float
	float IVA = 21*A/100;
	float IMP = IVA + A;
    System.out.println("IVA 21% "+ IVA +"€");
    System.out.println("Cantidad a pagar " + IMP +"€");

  }
}

File: UF1/M3-UF1-practica-5/ejercicio11.py
#! /usr/bin/env python
# encoding: utf-8
# Programita 1

print "Mates varias"
A = raw_input("Introduce el precio del producto:")
A= float(A) 
IVA = 21*A/100
IMP = IVA + A
print "IVA 21% ", IVA,"€"
print "Cantidad a pagar", IMP,"€"

